paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentMethodsUI.kt

```diff
@@ -71,9 +73,8 @@ internal fun PaymentMethodsUI(
             // TODO: userScrollEnabled will be available in compose version 1.2.0-alpha01+
```

Comment:
Since we're here, it looks like this is using the LazyListState to block scrolling because `userScrollEnabled` was not available, but now you could use that as a parameter to LazyRow below and delete lines 56-62